### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
-language: objective-c
+language: generic
```
Comment (ChrisBarker-NOAA) [inline .travis.yml:4]: so I take it we no longer need to pretend we're using objective-c to get an OS-X build machine?

File: README.md [hunk 2]
```diff
@@ -14,31 +14,32 @@ Summary: A Python API to utilize data written using the unstructured grid UGRID
 Installing pyugrid
```
Comment (ChrisBarker-NOAA) [inline README.md:14]: Does this need to be updated by hand? Or is there some bot that can do it?

**Review by ChrisBarker-NOAA — COMMENTED** [review]
just wondering about the process here. I've got other recipes to deal with...

File: README.md (hunk 2 above)
Comment (ocefpaf) [reply README.md:14]: We use `conda-smithy` for the feedstocks updates.

**Pull request merged**